packages/radio-player/test/radio-player.test.js

```diff
@@ -866,4 +866,207 @@ describe('Radio Player', () => {
       expect(quickSearches[2].displayText).to.equal('baz');
     });
   });
+
+  describe('Events', () => {
+    it('emits playbackRateChanged event', async () => {
+      const el = await fixture(html`
+        <radio-player></radio-player>
+      `);
+
+      const event = new CustomEvent('foo', {
+        detail: { playbackRate: 1.25 },
+      });
+      setTimeout(() => { el.changePlaybackRate(event); });
+      const response = await oneEvent(el, 'playbackRateChanged');
+      expect(response.detail.playbackRate).to.equal(1.25);
+    });
+
+    it('emits volumeChanged event', async () => {
+      const el = await fixture(html`
+        <radio-player></radio-player>
+      `);
+
+      const event = new CustomEvent('foo', {
+        detail: { volume: 0.5 },
+      });
+      setTimeout(() => { el.volumeChanged(event); });
+      const response = await oneEvent(el, 'volumeChanged');
+      expect(response.detail.volume).to.equal(0.5);
+    });
+
+    it('emits jumpBackButtonPressed event', async () => {
```

Comment:
**Suggestion**: For some of these, you can abstract the code to a generic function and just pass in the event name in test. So with something like:

```javascript
const testEventEmitter = (eventName) => {
  it(`emits ${eventName} event`, async () => {
    const el = await fixture(html`
      <radio-player></radio-player>
    `);

    setTimeout(() => { el.canplay(); });
    const response = await oneEvent(el, eventName);
    expect(response).to.exist;
  });
};
```

you could then write `['jumpBackButtonPressed', 'jumpForwardButtonPressed', ...].forEach(testEventEmitter);`

Reply:
Inside the `setTimeout`, I'm doing different things for each event, ie `setTimeout(() => { el.canplay(); });` produces a `canplay` event. 

How would you do that with this approach?

Reply:
Yeah, sorry. The actual code should call the method like `setTimeout(() => { el[eventName]() })`. Then you can convert a number of these to use this same pattern.

Reply:
If the method names differ, you could also use an array of objects instead, so `{ name: 'jumpFOrwardButtonPressed', method: 'forwardButtonHandler' }`